server/views/challenges/showBonfire.jade

```diff
@@ -80,6 +80,7 @@ block content
                                   = phrase
                       else
                           a#next-challenge.btn.btn-lg.btn-primary.btn-block(href="/challenges/next-challenge?id="+id) Go to my next challenge (ctrl + enter)
+                      #show-solution.animated.fadeIn.btn.btn-lg.btn-primary.btn-block See the recommended solution
```

Comment:
Let's address this separately as discussed in the other issue for showing a recommended solution side by side.
You need to remove the buttons for now. Leave a comment on the other issue, if confirmed, make a separate PR